bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/inbox/InboxFilterCriteria.java

```diff
@@ -26,61 +27,137 @@
     private final String bindingId;
     private final ThingTypeUID thingTypeUID;
     private final ThingUID thingUID;
+    private final String representationValue;
```

Comment:
Leaving InboxFilterCriterla completely unchanged would mean to have this odd constructors also in the 1.0 release eventually. And how to introduce the predicates then? May be I didn't get the idea of predicates yet...